test/components/selectionBoxLayerTests.ts

```diff
@@ -48,6 +48,43 @@ describe("Interactive Components", () => {
         svg.remove();
       });
 
+      it("generates the correct clipPath", () => {
+        sbl.renderTo(svg);
+
+        TestMethods.verifyClipPath(sbl);
+        let clipRect = (<any> sbl)._boxContainer.select(".clip-rect");
+        assert.strictEqual(TestMethods.numAttr(clipRect, "width"), SVG_WIDTH, "the clipRect has an appropriate width");
+        assert.strictEqual(TestMethods.numAttr(clipRect, "height"), SVG_HEIGHT, "the clipRect has an appropriate height");
+        svg.remove();
+      });
+
+      it("updates the clipPath reference when render()-ed", () => {
```

Comment:
Again, since this is a straight-up copy + paste remove it from the `dragBoxLayerTests`